corehq/apps/style/tests/test_compress_command.py

```diff
@@ -66,6 +67,7 @@ def _is_b3_base_template(self, template):
 
         return False
 
+    @attr("slow")
     def test_compress_offline(self):
```

Comment:
This test can now be skipped with `./manage.py test --settings=testsettings -a '!slow'`